src/OneExplorer.ts

```diff
@@ -152,29 +177,131 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
+      {section: 'one-codegen', key: 'command', grepper: this.grepTargetInCommand},
+    ];
+
+    let intermediates: string[] = [];
+    for (let loc of targetLocator) {
+      let confSection = config[loc.section as keyof typeof config];
+      let confKey = confSection ? confSection[loc.key as keyof typeof config] : undefined;
+      if (confKey) {
+        const targets = loc.grepper(confKey);
+        for (let target of targets) {
+          if (intermediates.includes(target) === false) {
+            intermediates.push(target);
+          }
+        }
+      }
+    }
+
+    return intermediates;
+  };
+
+  /**
+   * compare paths by normalization
+   * NOTE that '~'(home) is not supported
+   * TODO(dayo) support '~'
+   * TODO(dayo) extract file-relative functions as another module
+   */
+  private comparePath(path0: string, path1: string): boolean {
+    const absPath0 = path.resolve(path.normalize(path0));
+    const absPath1 = path.resolve(path.normalize(path1));
+    return absPath0 === absPath1;
+  }
+
   /**
-   * Search .cfg files in the same directory of the node
-   *
-   * NOTE It assumes 1-1 relation for model and config
-   *
-   * TODO(dayo) Support N-N relation
-   * TODO(dayo) Search by parsing config file's model entry (Currently model name and cfg name must
-   * match)
+   * Search .cfg files in the same directory
    */
   private searchPairConfig(node: Node) {
+    console.assert(node.type === NodeType.baseModel);
+
     const files = fs.readdirSync(node.parent);
 
-    const extSlicer = (fileName: string) => {
-      return fileName.slice(0, fileName.lastIndexOf('.'));
-    };
+    for (const fname of files) {
+      const fpath = path.join(node.parent, fname);
+      const fstat = fs.statSync(fpath);
+
+      if (fstat.isFile() && fname.endsWith('.cfg')) {
+        const parsedInputPath = this.parseInputPath(fpath, node.path);
+        if (parsedInputPath) {
+          const fullInputPath = path.join(node.parent, parsedInputPath);
+          if (this.comparePath(fullInputPath, node.path)) {
+            const pairNode = new Node(NodeType.config, [], vscode.Uri.file(fpath));
+            this.searchChildModels(pairNode);
+            node.childNodes.push(pairNode);
+          }
+        }
+      }
+    }
+  }
+
+  /**
+   * Search specified intermediate model files in the same directory
+   */
+  private searchChildModels(node: Node) {
+    console.assert(node.type === NodeType.config);
+    console.log('searchChildModels');
+    const files = fs.readdirSync(node.parent);
 
     for (const fname of files) {
       const fpath = path.join(node.parent, fname);
       const fstat = fs.statSync(fpath);
 
-      if (fstat.isFile() && fname.endsWith('.cfg') && (extSlicer(fname) === extSlicer(node.name))) {
-        const pairNode = new Node(NodeType.config, [], vscode.Uri.file(fpath));
-        node.childNodes.push(pairNode);
+      if (fstat.isFile() && (fname.endsWith('.circle') || fname.endsWith('.tvn'))) {
```

Comment:
File extention could be different from backend to backend. Rather than using string `.tvn`, it would be better to use some backend API to get file extention string by asking backend API.
Please refer to the approach in #549.

Reply:
Yeap. I think that this type of functions are better to be extracted out as a module, I don't expected OneExplorer to parse things, but as it need to look into config file to grep target models, parser is needed. I have told this to @llFreetimell before.

I also plan to extract this as a function, but later.

Reply:
Hmm...? What I talked with @dayo09 was about parsing `.cfg`, not about getting file extension list...? @.@

Reply:
> I think that this type of functions are better to be extracted out as a module,

@llFreetimell OneExplorer now looks into ini file and parse [one-codegen] config so on. I say this can be extracted out as a module.

Reply:
I am worried if you are against the idea - making another module to look into filesystem and parse config file and specify file extension list?

Reply:
> it would be better to use some backend API to get file extention string by asking backend API.

I understood like following.

```typescript
// before
if (fstat.isFile() && (fname.endsWith('.circle') || fname.endsWith('.tvn'))) {

// after (Executor.getExecutableExt is in #549)
if (fstat.isFile() && Executor.getExecutableExt().includes(fname.substring(fname.lastIndexOf('.')))
```

So I thought that this is irrelevant with ini file or parsing it.

Reply:
Currently your code will work but he also implies the case of 'other backends' which uses other ext formats.
The code above won't resolve here, as it gets 'executable ext' which are only .circle and .tvn.
I have temporarily made this to support .circle and .tvn as config child but actually if there are .tflite or .onnx, they should be displayed too.(furthur feature)

As I said, I am planning to make a function like `getChildExt`, of course.

Reply:
@dayo09 What I am expecting is
- ONE-vscode should not know `.tvn` because it is specific to triv backend.
- likewise, if backend dummy's final code has `.dummy` file extension, ONE-vscode should not know `.dummy`.

So as long as we don't have `.tvn` string and as long and this code can handle `.dummy`, it would be great.

Reply:
Given timeframe for internal release, I think we can delay this support later. 
@dayo09 Could you add `TODO Get `.tvn` file extension from backend` comment? Thanks.

Reply:
I understand. I will make a change on another PR.

Reply:
Oh sorry that I missed this comment. 

> Given timeframe for internal release, I think we can delay this support later.
> @dayo09 Could you add TODO Get .tvn file extension from backend comment? Thanks.
> 

I will make immediate chagne about that.